src/Illuminate/Foundation/helpers.php

```diff
@@ -780,3 +780,17 @@ function view($view = null, $data = [], $mergeData = [])
         return $factory->make($view, $data, $mergeData);
     }
 }
+
+if (! function_exists('loop')) {
```

Comment:
I thought of having a blade directive `@loop('isFirst')` `@loop('isLast')` but then it won't be available inside `<?php if(@loop('isLast'))) //; ?>` code.